tests/store/api/conftest.py

```diff
@@ -164,6 +171,56 @@ def fixture_max_nr_of_cases() -> int:
     return 50
 
 
+@pytest.fixture(name="store_failing_sequencing_qc")
+def fixture_store_failing_sequencing_qc(
+    case_id: str,
+    family_name: str,
+    bcl2fastq_flow_cell_id: str,
+    sample_id: str,
+    ticket_id: str,
+    timestamp_now: dt.datetime,
+    helpers,
+) -> Generator[Store, None, None]:
+    """Populate a store with a Fluffy case, with a sample that has been sequenced on two flow cells."""
+    store = Store(uri="sqlite:///")
```

Comment:
I think you can use `base_store` here to not have to create it on your own

Reply:
fixed